Clients/src/presentation/components/Table/index.tsx

```
@@ -141,19 +143,26 @@ const BasicTable = ({
                     borderBottom: "1px solid #EEEEEE",
-                    cursor: row.handleClick ? "pointer" : "default",
+                    cursor: onRowClick ? "pointer" : "default",
                     "&:hover": {
                       backgroundColor: theme.palette.background.accent,
                     },
                   }}
                   key={row.id}
-                  onClick={row.handleClick ? row.handleClick : null}
+                  onClick={() => {
+                    console.log(`Row clicked: ${row.id}`); 
+                    onRowClick && onRowClick(row.id);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)